src/main/java/org/group23/TextQuestion.java

```diff
@@ -1,18 +1,35 @@
 package org.group23;
 
-public class TextQuestion extends Question{
+import jakarta.persistence.ElementCollection;
+import jakarta.persistence.Entity;
 
-    private String textResponse;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+public class TextQuestion extends Question {
+
+    @ElementCollection
+    private List<String> answers;
```

Comment:
Is this how we're storing user answers to the question? If so have you tried retrieving all the answers to a given question manually? (HTTP) If it works that's really convenient!